src/lib.rs

```diff
@@ -13,6 +16,11 @@ pub use tsify_macros::*;
 #[cfg(feature = "wasm-bindgen")]
 use wasm_bindgen::{JsCast, JsValue};
 
+#[cfg(feature = "json")]
+pub type Error = serde_json::Error;
+#[cfg(feature = "js")]
+pub type Error = serde_wasm_bindgen::Error;
+
```

Comment:
We actually don't need a NAND filter on these two features any more if we kill the into_wasm_abi/from_wasm_abi macro flags. But I suspect that's a #[deprecated] for a while thing as this crate has been around for a while.